packages/client/src/views/game-view.tsx

```diff
@@ -33,9 +33,11 @@ export const GameView = ({ levelId }: GameViewProps) => {
   const nextLevel = useGoToNextLevel(levelId);
 
   const [game, setGame] = useState<GameClass>();
+  const [isFlaggedAlready, setIsFlaggedAlready] = useState(false);
```

Comment:
**thought**: `flagged, setFlagged` would be enough IMO, but `isFlaggedAlready` or `isAlreadyFlagged` works too